config/jobs/kubernetes/sig-auth/serviceaccount-admission-controller-migration-config.yaml

```diff
@@ -0,0 +1,28 @@
+periodics:
+- name: sig-auth-serviceaccount-admission-controller-migration
+  annotations:
+    testgrid-dashboards: sig-auth-gce
+    testgrid-tab-name: gce
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+  decorate: true
+  interval: 2h
```

Comment:
Oh, I forgot, we want to set up notifications for this job. This can be individuals until we get the job green, then should transition to the sig-auth list. See use of testgrid-alert-email in other jobs for examples, and include the mailing list transition in the follow-up tracking issue

Reply:
done